1-JavaScript/Week3/homework/ex6-totalCost.js

```diff
@@ -20,12 +20,22 @@ instead!
    Use `console.log` to display the result.
 -----------------------------------------------------------------------------*/
 const cartForParty = {
-  // TODO complete this function
+  beers: 1.1,
+  chips: 2.9,
+  cola: 2.3,
+  olives: 2.7,
+  bacon: 2, //11.00
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
-  // TODO replace this comment with your code
+function calculateTotalPrice(cart) {
```

Comment:
Hi Veronica - although this logic works fine but you could improve it slightly. Firstly, you should try to avoid using `for-in` loops for iterating over objects. Try to use `Object.keys()` instead. The difference between the two can be found out on google, it's something to do regarding iterating over parent object properties. Secondly, if you use Object.keys, you could also look at using the array's reduce method. Something like below:

```
function calculateTotalPrice(cart) {
  const total = Object.keys(cart).reduce((acc, elem) => acc + cart[elem], 0);
  return `Total: ${total}`;
}
```